src/core/algorithms/md/hymd/hymd.h

```diff
@@ -32,6 +32,7 @@ class HyMD final : public MdAlgorithm {
     bool prune_nondisjoint_ = true;
     std::size_t max_cardinality_ = -1;
     config::ThreadNumType threads_;
+    bool no_levels_ = false;
```

Comment:
Use a `BETTER_ENUM` option `level_definition` with options `cardinality` and `lattice` and the function is chosen based on this enum, empty function means no levels (`lattice`).